2/contrib/jenkins/kube-slave-common.sh

```diff
@@ -26,6 +26,8 @@ export JNLP_PORT=${JNLP_PORT:-50000}
 
 NODEJS_SLAVE=${NODEJS_SLAVE_IMAGE:-registry.access.redhat.com/openshift3/jenkins-slave-nodejs-rhel7}
 MAVEN_SLAVE=${MAVEN_SLAVE_IMAGE:-registry.access.redhat.com/openshift3/jenkins-slave-maven-rhel7}
+DOTNET10_SLAVE=${NODEJS_SLAVE_IMAGE:-registry.access.redhat.com/openshift3/jenkins-slave-dotnet10-rhel7}
+DOTNET11_SLAVE=${NODEJS_SLAVE_IMAGE:-registry.access.redhat.com/openshift3/jenkins-slave-dotnet11-rhel7}
```

Comment:
if there's no centos image available, then the logic below that adds the dotnet slaves to the config needs to be conditionalized to not do so when this is running inside the centos jenkins image.